core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -104,11 +93,9 @@ public Configuration(Properties props) {
 
   /**
    * Default constructor.
-   *
-   * Most clients will call this constructor to allow default loading of properties to happen.
    */
   public Configuration() {
```

Comment:
I think there is actually value of replacing all the references in this PR ... that way we know that the default / worker / master / client configuration is used where it should be used